engine/src/main/java/org/hibernate/search/engine/logging/impl/Log.java

```diff
@@ -166,4 +166,17 @@ SearchException relativeFieldNameCannotContainDot(String relativeFieldName,
 			value = "Multiple calls to createAccessor() for the same field definition."
 					+ " You must call createAccessor() exactly once.")
 	SearchException cannotCreateAccessorMultipleTimes(@Param EventContext context);
+
+	@Message(id = ID_OFFSET_2 + 25,
+			value = "Cannot call ifSupported(...) after otherwise(...)."
```

Comment:
I think you mean `orElse()` here instead of `otherwise()`. At least from a user facing POV, that's what he will see.

Maybe we should be consistent to avoid this issue?

Reply:
Right, I forgot these when I renamed `otherwise()` to `orElse()` while working on it. Which I did to be consistent with `Optional#orElse`.

Fixed.